certified-connectors/Eagle/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2577 @@
+        ],
+        "description": "This action will update an exising EBMS product."
+      }
+    },
+    "/INVENTRE": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default"
+          }
+        },
+        "summary": "Get Product Category ID",
+        "description": "This will get all the Product Categories.",
+        "operationId": "GetProductCategoryID",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/TASK": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create a new task (task module required)",
+        "operationId": "CreateTask",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "TYPE": {
+                  "type": "string",
+                  "description": "Type",
+                  "title": "Task Type",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetTaskType",
+                    "value-path": "TYPE",
+                    "value-collection": "value",
+                    "value-title": "TYPE",
+                    "parameters": {}
+                  },
+                  "x-ms-visibility": "important"
+                },
+                "CUST_ID": {
+                  "type": "string",
+                  "description": "Customer Id",
+                  "title": "Customer Id"
+                },
+                "DESCR": {
+                  "type": "string",
+                  "description": "Description",
+                  "title": "Description"
+                },
+                "EMP_ID": {
+                  "type": "string",
+                  "description": "Worker ID",
+                  "title": "Worker ID"
+                },
+                "START_DATE": {
+                  "type": "string",
+                  "description": "Start date in the format yyyyMMdd",
+                  "title": "Start date"
+                },
+                "START_TIME": {
+                  "type": "string",
+                  "description": "Start time in 12 or 24hr format",
+                  "title": "Start time"
+                },
+                "END_DATE": {
+                  "type": "string",
+                  "description": "End date in the format yyyyMMdd",
+                  "title": "End date"
+                },
+                "END_TIME": {
+                  "type": "string",
+                  "description": "End time  in 12 or 24hr format",
+                  "title": "End time"
+                },
+                "DUE_BY": {
+                  "type": "string",
+                  "description": "Due date  in the format yyyyMMdd",
```

Comment:
@doniyorniazov please make sure that descriptions and titles are grammatically correct and are free of typographical errors. There's an extra whitespace in this description.